README.md

```diff
@@ -31,26 +31,59 @@ mvn package
 Create the sqlite databases
 ```
 echo "CREATE TABLE players (id INT PRIMARY_KEY, name VARCHAR(20), password VARCHAR(50), email VARCHAR(50), r4 INT, r5 INT,r6 INT, r7 INT,r8 INT, rating real default 1000, boost real default 750, ratedgames int default 0, maxrating real default 1000, ratingage real default 0, ratingbase int default 0, unrated int default 0, isbot int default 0, fatigue text default '{}');" | sqlite3 target/players.db
-echo "CREATE TABLE games (id INTEGER PRIMARY KEY, date INT, size INT, player_white VARCHAR(20), player_black VARCHAR(20), notation TEXT, result VARCAR(10), timertime INT DEFAULT 0, timerinc INT DEFAULT 0, rating_white int default 1000, rating_black int default 1000, unrated int default 0, tournament int default 0, komi int default 0, pieces int default -1, capstones int default -1, rating_change_white int default 0, rating_change_black int default 0);" | sqlite3 target/games.db
+echo "CREATE TABLE games (id INTEGER PRIMARY KEY, date INT, size INT, player_white VARCHAR(20), player_black VARCHAR(20), notation TEXT, result VARCAR(10), timertime INT DEFAULT 0, timerinc INT DEFAULT 0, rating_white int default 1000, rating_black int default 1000, unrated int default 0, tournament int default 0, komi int default 0, pieces int default -1, capstones int default -1, rating_change_white int default 0, rating_change_black int default 0, extra_time_amount int default 0, extra_time_trigger int default 0);" | sqlite3 target/games.db
```

Comment:
Does this create table command modifies the existing table? From reading the script it looks like it would lose all the data that currently exists.

Reply:
I believe this may be implementation dependant, some SQL Servers support `CREATE TABLE <name IF NOT EXIST` however for sqlite3 I'm getting `Parse error: table games already exists` when running this.

These are instructions for devs anyways. I've only changed the line because it got out of sync with the DB Structure and the `create_databases.sh` script which **DOES** delete the databases before recreating them.